config/routes.rb

```diff
@@ -9,4 +9,8 @@
     resources :posts, controller: "users/posts", only: [:index]
   end
 
+  resources :posts, :comments do
+    resources :comments
```

Comment:
This creates a lot of routes that aren't getting used. Can you use `only` or `except` to only create the routes that exist?